.github/workflows/tox.yml

```diff
@@ -20,12 +20,12 @@ jobs:
 
     steps:
       - name: Check out the repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Update the GH action to the latest version - no idea what has changed, but there may be some performance tweaks in there?